tests/ibc_pfm_with_grace_period_test.go

```diff
@@ -37,10 +36,13 @@ func TestIBCPFMWithGracePeriod_EVM(t *testing.T) {
 	dymintTomlOverrides["gas_prices"] = "0adym"
 	dymintTomlOverrides["empty_blocks_max_time"] = "3s"
 
-	modifyGenesisKV := append(dymensionGenesisKV, cosmos.GenesisKV{
-		Key:   "app_state.rollapp.params.dispute_period_in_blocks",
-		Value: "60",
-	})
+	modifyGenesisKV := append(
+		dymensionGenesisKV,
+		cosmos.GenesisKV{
+			Key:   "app_state.rollapp.params.dispute_period_in_blocks",
+			Value: fmt.Sprint(BLOCK_FINALITY_PERIOD),
```

Comment:
the old value is 60 but `BLOCK_FINALITY_PERIOD` is 30, does this effect with our test @anhductn2001

Reply:
I think it's not @ThanhNhann